composites/Plugin/ContentAnalysis/reducers/tests/contentAnalysisReducerTest.js

```diff
@@ -0,0 +1,40 @@
+import { UPDATE_SEO_RESULT, UPDATE_READABILITY_RESULT } from "../../actions/contentAnalysis";
+import { contentAnalysisReducer } from "../../reducers/contentAnalysisReducer";
+
+jest.mock( "../../reducers/contentAnalysis/readabilityResultsReducer", () => {
+	return {
+		readabilityResultsReducer: jest.fn( () => {
+			return { name: "readabilityResultsReducer" };
+		} ),
+	};
+} );
+
+jest.mock( "../../reducers/contentAnalysis/keywordResultsReducer", () => {
+	return {
+		keywordResultsReducer: jest.fn( () => {
+			return { name: "keywordResultsReducer" };
+		} ),
+	};
+} );
+
+test( "content analysis reducer with the UPDATE_SEO_RESULT action", () => {
```

Comment:
I admit it is a kind of silly test, but It tests whether the correct reducers are called from the contentAnalysisReducer. I've removed the test with the UPDATE_READABILITY_RESULT because it is not necessary to test for each of the actions indeed. One is sufficient.

What the initial state looks likes in those called reducers is the underlying reducers' concern. No initial state is set in contentAnalysisReducer.js.